command/debug.go

```diff
@@ -441,29 +553,29 @@ func (c *DebugCommand) preflight(rawArgs []string) (string, error) {
 			c.flagOutput = strings.TrimSuffix(c.flagOutput, ".tar.gz")
 			c.flagOutput = strings.TrimSuffix(c.flagOutput, ".tgz")
 		case err != nil:
-			return "", fmt.Errorf("unable to stat file: %s", err)
+			return "", nil, fmt.Errorf("unable to stat file: %s", err)
 		default:
-			return "", fmt.Errorf("output file already exists: %s", dstOutputFile)
+			return "", nil, fmt.Errorf("output file already exists: %s", dstOutputFile)
 		}
 	}
 
 	// Stat check the directory to ensure we don't override any existing data.
-	_, err = os.Stat(c.flagOutput)
+	_, err := os.Stat(c.flagOutput)
 	switch {
 	case os.IsNotExist(err):
 		err := os.MkdirAll(c.flagOutput, 0o755)
 		if err != nil {
-			return "", fmt.Errorf("unable to create output directory: %s", err)
+			return "", nil, fmt.Errorf("unable to create output directory: %s", err)
 		}
 	case err != nil:
-		return "", fmt.Errorf("unable to stat directory: %s", err)
+		return "", nil, fmt.Errorf("unable to stat directory: %s", err)
 	default:
-		return "", fmt.Errorf("output directory already exists: %s", c.flagOutput)
+		return "", nil, fmt.Errorf("output directory already exists: %s", c.flagOutput)
 	}
 
 	// Populate initial index fields
-	c.debugIndex = &debugIndex{
-		VaultAddress:           client.Address(),
+	debugIndex := &debugIndex{
+		// VaultAddress:           client.Address(),
```

Comment:
why are we removing this

Reply:
This is the "base" debugIndex which is common to all nodes.  Later we use it as the basis for node-specific debugIndexes which include the address.